setup.py

```diff
@@ -137,6 +163,7 @@ def run(self):
     cmdclass={
         "docs_api": GenerateSphinxApidoc,
         "docs": BuildDoc,
```

Comment:
If `docs_versioned` is the go-to command, perhaps we can remove the commands `docs_api` `docs`, or is there some use-case for them still? 

I don't think we'll need to API on it's own, but perhaps unversioned docs, although we likely wouldn't want to publish those over the versioned ones so I'm not sure we need to keep it

Reply:
We can't full remove the `docs_api` and `docs` rules, because the `docs_versioned` calls them. I couldn't find a way to trigger other rules from within the setup.py, other than using `subprocess`. (there is `distutils.core.run_setup` but I'd did not work for me),

Would it make sense to rename them? So there are internal `_docs` and `_api_docs` commands, and then rename `docs_versioned` to just `docs`?

Reply:
just pushed a new version with this

Reply:
That's great!